aten/src/ATen/functorch/PyTorchOperatorHacks.cpp

```diff
@@ -188,7 +188,8 @@ bool is_fused_kernel_acceptable(const Tensor& input, double p) {
 template<bool inplace>
 Tensor& multiply(Tensor& input, const Tensor& noise) {
   static_assert(inplace, "Wrong multiply overload triggered in Dropout.cpp");
-  return input.mul_(noise);
+  // this is now broken but we don't care
+  return input;  // input.mul_(noise);
```

Comment:
hmmm why is this broken? (This looks like it's just the normal tensor API: `at::Tensor::mul(const Tensor& other)`